test_perf.sh

```
@@ -0,0 +1,44 @@
+cat << 'INNER_EOF' > src/test/java/com/sayai/record/service/CrawlingServiceMockPerfTest.java
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)